pkg/sql/rowexec/backfiller.go

```diff
@@ -268,6 +268,10 @@ func GetResumeSpans(
 	// Find the job.
 	var jobID int64
 	if len(tableDesc.MutationJobs) > 0 {
+		// TODO (lucy): We need to get rid of MutationJobs. This is the only place
```

Comment:
Ugh, yeah, I'd forgotten this bit of code, and how unfortunate its use of jobs was. I've been wanting to rework this and how we checkpoint/load spans constantly in the backfiller and this is yet another bit of motivation.